### PR TITLE
Centos8/stream supported

### PR DESCRIPTION
**Pull request by Jukepoika**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@
       msg: "collectd_set_graphite_prefix is deprecated. See defaults/main.yml"
 
   - include_tasks: redhat.yml
-    when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "7"
```
Comment (Kallio) [inline tasks/main.yml:10]: is this check syntax supported on older ansbile versions?

Comment (Jukepoika) [reply tasks/main.yml:10]: If Centos8/stream support is not needed, it is possible to stick on older commits. It is also possible to stick on legacy syntax, I guess? @khappone @Aegil

Comment (khappone) [reply tasks/main.yml:10]: If this syntax works, and the code works for 7/8, Jukkas suggestion is good.

Comment (Jukepoika) [reply tasks/main.yml:10]: I can confirm this syntax works with Ansible 2.9 and for Centos 7/8.

**Pull request merged**